### PR TITLE
enhancement/denham_springs_pd

### PR DESCRIPTION
**Pull request by ayyubibrahimi**

### CODE REVIEW

File: data/datavalid.yml [hunk 1]
```diff
@@ -158,6 +158,9 @@ schemas:
       - name: investigator_uid
       - name: investigator_rank
       - name: shift_supervisor
+      - name: allegation_desc
+        description: >
+          text description of the allegation
```
Comment (pckhoi) [inline data/datavalid.yml:163]: @rajivsinclair  Should we rename this field to `summary` to keep it consistent with previous work à la CPDP?

Comment (ayyubibrahimi) [reply data/datavalid.yml:163]: I think that `allegation_summary` or `allegation_desc` is more appropriate as `summary` seems vague.

Comment (rajivsinclair) [reply data/datavalid.yml:163]: I think `allegation_desc` or `allegation_summary` is okay here.

Comment (pckhoi) [reply data/datavalid.yml:163]: that's fine then

**Review by pckhoi — APPROVED** [review]
LGTM

**Pull request merged**